apps/transport/lib/transport_web/api/controllers/datasets_controller.ex

```diff
@@ -73,6 +73,7 @@ defmodule TransportWeb.API.DatasetController do
       "resources" => Enum.map(dataset.resources, &transform_resource/1),
       "aom" => transform_aom(dataset.aom),
       "type" => dataset.type,
+      "publisher" => %{"organization" => dataset.organization},
```

Comment:
the publisher might be an organization or a user.
for that I would use `Transport.DatasetView.get_name` function

Reply:
oups indeed, I did not know this.
this thus brings some new question.
Do we want to make this difference in the API ?

# solution 1: hide
{"publisher": {"name": "<some organisation>"}
{"publisher": {"name": "<firstname lastname>"}

# solution 2: similar to the model
{"publisher": {"organization": "<some organisation>"}
{"publisher": {"user": "<firstname lastname>"} (or something more structured)

# solution 3: easier to use ?
{"publisher": {"name": "<some organisation>", "type"="organization"}
{"publisher": {"name": "<firstname lastname>", "type"="user"}

# solution 4: hybrid, more complex
{"publisher": {"name": "<some organisation>", "organization": {"id: "<datagouv_id>"}}}
{"publisher": {"name": "<firstname lastname>", "user": {"id": "<datagouv_id>", first_name: "<firstname>", last_name": "<last_name>"}}}
+ maybe a "type" field like in solution 3 for better self documentation

(but I don't see the datagouv id in the model)

what to you think ?
I feel the fourth one is better, but maybe it's overkill (and we need an id)

Reply:
I prefer solution 3

Reply:
hum are you sure the publisher can be either an `organization` or a `owner` ?
I do not see an `owner` field, apart from some helpers...

Reply:
Yup, it happens when the user makes a mistake and doesn't create it with an organization...

Reply:
but the `dataset` has no field `owner` 🤔

Reply:
So you think we should read the `owner` when importing the datasets?

Reply:
as you want. If we do this I think it would be nice to also import it's datagouv_id (and the url ?)

Do is it useful ?